### DISCUSSION TITLE
A few questions about leather engine

### DISCUSSION THREAD

**Discussion by redstonemaster60** [Leather128/LeathersFunkinEngine#225] [Q&A] [answered]
1. If you import a chart from Pysch Engine into Leather Engine, does it still work or does Leather Engine have a different way that charts work?
2. Is there a way to get events or modcharts from Pysch Engine to run on Leather Engine?
3. Is there any documentation of what the PlayState variables are and if not, where can I find them?

Thanks if you know either of these.

**Comment by Leather128** [comment, marked as answer]
1. Yes it should as long as you aren't using a really old version of Psych (because sometimes those versions put events in regular chart data)
2. You'd have to re code them manually, but yes (because events from Psych charts get detected in Leather Engine).
3. Nope, just look at [PlayState.hx](https://github.com/Leather128/LeatherEngine/blob/main/source/states/PlayState.hx) to see all accessible variables for PlayState.

**Reply by redstonemaster60** [reply]
alr thx
